scripts/Render-SasEnglishReport.ps1

```
@@ -0,0 +1,237 @@
+    'probe_target_file',
+    'ports_requested',
+    'target_count',
+    'network_preflight_csv',
+    'newly_reachable_count',
+    'still_silent_count',
+    'stale_or_conflicting_count',
+    'persistent_silent_time_diverse_count',
+    'plateau_detected'
+)) {
+    if ($summary.PSObject.Properties.Name -contains $name) {
+        $results += "- $name: $($summary.$name)"
```

PSScriptAnalyzer (1.25.0)

[error] Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

(InvalidVariableReferenceWithDrive)